src/views/Rules/ListRules/index.jsx

```diff
@@ -415,6 +415,28 @@ function ListRules(props) {
           // diff viewer
           height += diffedProperties.length * 21 + theme.spacing(5);
         }
+
+        if (Object.keys(rule.scheduledChange.required_signoffs).length > 0) {
+          const requiredRoles = Object.keys(
+            rule.scheduledChange.required_signoffs
+          ).length;
+          const nSignoffs = Object.keys(rule.scheduledChange.signoffs).length;
+
+          // Add space for the "Requires Signoff From" title
+          height += theme.spacing(4);
+
+          // Add space for each row of required roles
+          height += theme.spacing(4 * requiredRoles);
+
+          // Signoffs and required roles are beside one another, so if there's
+          // the same number or fewer signoffs than required roles, we don't
+          // need to add extra space for them. If the signoffs are more
+          // mnumerous than the number of required roles, we need more space
+          // for the extras.
+          if (nSignoffs > requiredRoles) {
+            height += theme.spacing(5 * (nSignoffs - requiredRoles));
+          }
+        }
```

Comment:
I think the heights need to be updated. Navigating to http://localhost:9000/rules#scId=21 shows cards with no spaces in between :



![Screen Shot 2019-07-23 at 10 28 43 AM](https://user-images.githubusercontent.com/3766511/61720377-ad5d7a80-ad34-11e9-8665-bf6fcc0625df.png)

Reply:
Hm, indeed. Once we figure out the other issue I'll take a look at this.